sessions/class-01/exercise/start.js

```diff
@@ -81,4 +111,10 @@ export function getFirstAndLastNames(name) {}
  * @param {any} def Default value.
  * @returns {any} Property value.
  */
-export function getValue(obj, key, def) {}
+export function getValue(obj, key, def) {
+	if (hasValue(obj)){
+		return obj(key);
```

Comment:
When you are accessing a property of an object, where the property name is a variable, use the square bracket notation.

```js
const prop = 'something';
someObj[prop];
```

Doing

```js
someObj(prop);
```

Is basically calling a function `someObj` with `prop` as first parameter. But since `someObj` is an object, and not a function, this code will throw.

Also this was a tricky question. See this

https://github.com/swashata/multidots-frontend-dev-course-plan/pull/1#discussion_r251199647